db/migrate/20190725113200-create-unscored-table-and-triggers.js

```diff
@@ -0,0 +1,102 @@
+'use strict'
+
+// Table Names
+const UNSCORED_TABLE_NAME = 'unscored'
+const EVENTS_TABLE_NAME = 'events'
+const SCORES_TABLE_NAME = 'scores'
+
+// Postgres Functions
+const ADD_UNSCORED_FUNCTION = 'add_unscored_uri()'
+const ADD_UNSCORED_FUNCTION_SQL = `CREATE OR REPLACE FUNCTION ${ADD_UNSCORED_FUNCTION}
+RETURNS TRIGGER AS
+$$
+BEGIN
+  INSERT INTO ${UNSCORED_TABLE_NAME}(uri)
+  SELECT NEW.uri
+  WHERE NOT EXISTS (
+    SELECT uri FROM ${SCORES_TABLE_NAME} WHERE uri = NEW.uri
+  )
+  ON CONFLICT DO NOTHING;
+  RETURN NEW;
+END;
+$$ language plpgsql;`
+const RE_ADD_UNSCORED_FUNCTION = 're_add_unscored_uri()'
+const RE_ADD_UNSCORED_FUNCTION_SQL = `CREATE OR REPLACE FUNCTION ${RE_ADD_UNSCORED_FUNCTION}
+RETURNS TRIGGER AS
+$$
+BEGIN
+  INSERT INTO ${UNSCORED_TABLE_NAME}(uri)
+  SELECT OLD.uri
+  WHERE NOT EXISTS (
+    SELECT uri FROM ${SCORES_TABLE_NAME} WHERE uri = OLD.uri
+  )
+  ON CONFLICT DO NOTHING;
+  RETURN OLD;
+END;
+$$ language plpgsql;`
+const DEL_UNSCORED_FUNCTION = 'del_unscored_uri()'
+const DEL_UNSCORED_FUNCTION_SQL = `CREATE OR REPLACE FUNCTION ${DEL_UNSCORED_FUNCTION}
+RETURNS TRIGGER AS
+$$
+BEGIN
+  DELETE FROM ${UNSCORED_TABLE_NAME} WHERE uri = NEW.uri;
+  RETURN NEW;
+END;
+$$ language plpgsql;`
+
+// Postgres Triggers
+const INSERT_EVENT_TRIGGER = 'trigger_insert_event'
+const INSERT_EVENT_TRIGGER_SQL = `CREATE TRIGGER ${INSERT_EVENT_TRIGGER}
+AFTER INSERT ON ${EVENTS_TABLE_NAME}
+FOR EACH ROW
+WHEN (NEW.uri IS NOT NULL)
+EXECUTE PROCEDURE ${ADD_UNSCORED_FUNCTION};`
+const INSERT_UPDATE_SCORE_TRIGGER = 'trigger_insert_update_score'
+const INSERT_UPDATE_SCORE_TRIGGER_SQL = `CREATE TRIGGER ${INSERT_UPDATE_SCORE_TRIGGER}
+AFTER INSERT OR UPDATE ON ${SCORES_TABLE_NAME}
+FOR EACH ROW
+EXECUTE PROCEDURE ${DEL_UNSCORED_FUNCTION};`
+const DELETE_SCORE_TRIGGER = 'trigger_delete_score'
+const DELETE_SCORE_TRIGGER_SQL = `CREATE TRIGGER ${DELETE_SCORE_TRIGGER}
+AFTER DELETE ON ${SCORES_TABLE_NAME}
+FOR EACH ROW
+WHEN (OLD.uri IS NOT NULL)
+EXECUTE PROCEDURE ${RE_ADD_UNSCORED_FUNCTION};`
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface
+      .createTable(UNSCORED_TABLE_NAME, {
+        uri: {
```

Comment:
@Omicron7 looks like we forgot to update `unscored.js` to remove the `last_refreshed` column.